zapcore/level.go

```diff
@@ -53,6 +53,11 @@ const (
 
 	_minLevel = DebugLevel
 	_maxLevel = FatalLevel
+
+	// UnknownLevel is an invalid value for Level.
+	//
+	// Core implementations may panic if they see messages of this level.
+	UnknownLevel = _maxLevel + 1
```

Comment:
Can we reorganize these so that we have an invalid zero value? e.g.

```go
const (
  InvalidLevel = iota
  DebugLevel
  // ...
  FatalLevel

  _minLevel = DebugLevel
  _maxLevel = FatalLevel
)
```

Reply:
Wouldn't this be a breaking change because the integer value of DebugLevel (and everything else) would change?

Reply:
I lean towards not changing values of published constants, even if the line around whether it's a breaking change is a bit blurry. Such a change _will_ break someone, and we don't get much in exchange.